assets/src/scripts/components/dailyValueHydrograph/graph-brush.js

```diff
@@ -0,0 +1,97 @@
+import {brushX, brushSelection} from 'd3-brush';
```

Comment:
I am going to open a refactor ticket to see if some of this code which is very similar to hydrograph/graph-brush.js can be reusued and moved to d3-rendering. But that does not need to be done right now. Just FYI.